tool_collections/vcflib/vcfcheck/vcfcheck.xml

```diff
@@ -19,27 +19,27 @@
          <option value="history">History</option>
        </param>
        <when value="cached">
-         <param name="ref_file" type="select" label="Select reference genome">
-           <options from_data_table="fasta_indexes">
-             <!--<filter type="data_meta" key="dbkey" ref="input_bam" column="value"/>-->
-           </options>
-	   <validator type="no_options" message="A built-in reference genome is not available for the build associated with the selected input file"/>
-         </param>
-	 <param name="input_vcf" type="data" format="vcf" label="Select VCF dataset">
-	   <!-- Validators are commented to allow users apply too to any build. May need to be revised in the future
-		<validator type="unspecified_build" />
-		<validator type="dataset_metadata_in_data_table" table_name="fasta_indexes" metadata_name="dbkey" metadata_column="1" message="Sequences are not currently available for the specified build." />
-	   -->
-	 </param>
+            <param name="ref_file" type="select" label="Select reference genome">
+               <options from_data_table="fasta_indexes">
+                 <!--<filter type="data_meta" key="dbkey" ref="input_bam" column="value"/>-->
+               </options>
+                <validator type="no_options" message="A built-in reference genome is not available for the build associated with the selected input file"/>

```

Comment:
Indentation. This file should probably completely re-written using a coherent indentation.